content/blog/2025-07-10-keyspace-save-the-date.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "Save the Date August 28, 2025: Keyspace"
+description = "Join the Valkey crew on August 28, 2025 in Amsterdam for the first ever Valkey conference: Keyspace!"
+date = "2025-07-10 01:01:01"
+authors= [ "kyledvs"]
```

Comment:
There is another tag called trending you probably want to add, so it get's included in the sidebar on the blog page.